src/main/java/core/CoreDriver.java

```diff
@@ -0,0 +1,45 @@
+package core;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * This class defines set properties for WebDriver.
+ */
+public class CoreDriver {
+
+    public static WebDriver driver;
+
+    /**
+     * This class return a instance of WebDriver.
+     */
+    public static WebDriver getWebDriver(String browser) {
+        if (driver == null) {
+            if (browser.equals("CHROME")) {
```

Comment:
Maybe for a future PR you could try other options than conditional if; for example, it could be set in another class. But for now it is good

Reply:
Its True